src/data/polynomial/basic.lean

```diff
@@ -53,67 +54,68 @@ The embedding from `R` is called `C`. -/
 structure polynomial (R : Type*) [semiring R] := of_finsupp ::
 (to_finsupp : add_monoid_algebra R ℕ)
 
+localized "notation R`[X]`:9000 := polynomial R" in polynomial
```

Comment:
Is there anything that should have higher precedence? Should we be setting the precedence of the `R` too, perhaps defining this as `postfix`?

Reply:
I copied the precedence of notation for non zero divisors. That's the only combo notation I've seen so far, and these two values allow for talking about non zero divisors of polynomial R without any parentheses.

If we come up another usage we can play around with the precedence. I also think this works properly with the multiplicative opposite right now.